scripts/image_publisher.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env python3
```

Comment:
Is this file necessary, or was it just used for debugging?

Reply:
This is a demo. It is for debugging but it doesn't add any dependencies and I think it is worth keeping